pants-plugins/api_spec/rules.py

```diff
@@ -0,0 +1,259 @@
+# Copyright 2021 The StackStorm Authors.
```

Comment:
Was this really created in 2021?

Reply:
Yeah. I wrote this plugin in 2021. I can update the header though because it will be merged in 2023